scout/server/blueprints/cases/controllers.py

```diff
@@ -246,6 +249,7 @@ def case(store, institute_obj, case_obj):
         full_name = "{} ({})".format(panel_obj["display_name"], panel_obj["version"])
         case_obj["panel_names"].append(full_name)
 
+    case_obj["panels"] = sorted(case_obj.get("panels", []), key=lambda d: d["display_name"])
```

Comment:
👍 Ah, nice - the lambda application is clever: I would have predicted a crash on empty panels, but it looks all good!